org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/text/correction/ActionMessages.properties

```diff
@@ -28,6 +28,8 @@ GenerateConstructorsAction_label=Generate Constructors
 GenerateConstructorsAction_ellipsisLabel=Generate Constructors...
 GenerateDelegateMethodsAction_label=Generate Delegate Methods...
 GenerateFinalModifiersAction_label=Change modifiers to final where possible
+GenerateFinalModifiersAction_templateLabel=Change modifiers to final for ''{0}''
+GenerateFinalModifiersAction_selectionLabel=Change modifiers to final
```

Comment:
I'm not so sure about calling it `Change modifiers ...` . In fact what this is doing is adding a single modifier. We're not touching other modifiers (eg. static, private, etc.). To me, `Add final modifier to ...` makes a bit more sense but I see you're specific label also used in JDT.

for the case of a single instance, I would make it `Add final modifier for ''{0}''`
for the case of many changes, we can stick with `Change modifiers to final` (like you have already).